Basic_viewer/include/CGAL/Qt/Basic_viewer.h

```diff
@@ -1806,9 +1538,99 @@ class Basic_viewer_qt : public CGAL::QGLViewer
   bool clipping_plane_rendering = true; // will be toggled when alt+c is pressed, which is used for indicating whether or not to render the clipping plane ;
   float clipping_plane_rendering_transparency = 0.5f; // to what extent the transparent part should be rendered;
 
-  std::vector<std::tuple<Local_point, QString> > m_texts;
 };
 
+//------------------------------------------------------------------------------
+class QApplication_and_basic_viewer
+{
+public:
+  QApplication_and_basic_viewer(const CGAL::Graphics_scene& buffer,
+                                const char* title="CGAL Basic Viewer"):
+    m_application(nullptr),
+    m_basic_viewer(nullptr),
+    m_argc(1)
+  {
+    m_argv[0]=new char[strlen(title)+1];
+    memcpy(m_argv[0], title, strlen(title)+1);
```

Comment:
Why not passing to `std::string` which would avoid new/delete/memcpy

Reply:
That type is required because `m_argc` and `m_argv` are consumed by the constructor [`QApplication::QApplication(int &argc, char **argv)`](https://doc.qt.io/qt-6/qapplication.html#QApplication):
https://github.com/CGAL/cgal/blob/a30fd9b1b69e5e3d2e4f1e92526933d23b52e156/Basic_viewer/include/CGAL/Qt/Basic_viewer.h#L1567